test/browser/components/data-table-model-test.js

```diff
@@ -59,60 +60,9 @@ test('Components -> DataTableModal.render', t => {
   );
 
   t.equal(wrapper.find(DataTableModal).length, 1, 'should render DataTableModal data');
-  t.equal(wrapper.find(ReactDataGrid).length, 1, 'should render ReactDataGrid data');
+  t.equal(wrapper.find(Autosizer).length, 1, 'should render Autosizer');
   t.equal(wrapper.find(DatasetTabs).length, 1, 'should render DatasetTabs');
   t.equal(wrapper.find(DatasetModalTab).length, 1, 'should render 1 dataset-modal-catalog');
-  t.equal(wrapper.find('.react-grid-Row').length, 10, 'should render 10 rows');
```

Comment:
React-Window doesn't provide subnested elements when running tests

Reply:
what do you mean by that?

If we only stop at `Autosizer` then there is no point running this test. The test here is to make sure different types of data render correctly. Please put this test back. The mis display of boolean value would have been caught by this test

Reply:
It looks like when running the tests, rows are not rendered because react-window requires the viewport of the browser to determine which rows are to be visualized